internal/nix/source.go

```diff
@@ -26,27 +25,38 @@ func SourceNixEnv() error {
 		)
 	}
 
+	// Source the nix script that sets the environment, and print the environment
+	// variables that it sets, in a way we can parse.
+
+	// NOTE: Only use shell built-ins in this script so that we don't introduce
+	// any dependencies on an external binary.
+	script := heredoc.Docf(`
+		. %s;
+		echo PATH=$PATH;
+		echo NIX_PROFILEs=$NIX_PROFILES;
+		echo NIX_SSL_CERT_FILE=$NIX_SSL_CERT_FILE;
```

Comment:
Did you confirm that single user install only has these as well?

Reply:
Yeah. Those the three it always sets.

Technically there's one more case for single user install: sometimes, but not always, it sets `MANPATH` (for man pages). Should I handle that one too?